### ISSUE TITLE
prisma client should be genrrated before running api

### ISSUE THREAD

**Issue by kensac** [kensac/api-rust#20]
[https://github.com/Brendonovich/prisma-client-rust/issues/195](url)

**Comment by kensac** [comment]
This is going to be a massive deployment problem because of #19. Since we have to manually edit the app state file, that means we either write our own state extractor or we have to do some very hack code modification for every deployment, or maybe a possible alternative to the URL on every deployment. Just very hacky overall

**Comment by kensac** [comment]
since #19 is now solved, this looks like it should not be a problem. However, the issue will stay open until we have deployed and verified.

**Comment by kensac** [comment]
Added this step to the dockerfile. The only note here is that the Database URL needs to be passed as an arg when the dockerfile is run. I will continue to look for alternatives and whether we can get the database from the cloud run connection like we do for API-v3.

**Comment by jdurrell** [comment]
I agree that getting the database from the Cloud Run connection is probably the best way to go. According to [https://cloud.google.com/sql/docs/mysql/connect-run](https://cloud.google.com/sql/docs/mysql/connect-run), it looks like we can just set up the connection in the Cloud Run instance, and then connect via Unix socket. I'm pretty sure this is what v3 does already instead of using a GCP/Firebase SDK, so that connection method should port pretty cleanly. I guess I don't technically know if Prisma supports Unix sockets, but I don't see why they wouldn't?
